### ISSUE TITLE
Subscribe banner needs translation

### ISSUE THREAD

**Issue by lmclister** [lmclister/testtwf-shanghai-issues#11]
Currently:
SUBSCRIBE! to the W3C mailing List and Follow @ testthewebfwd and # testtwf

**Comment by zerlot** [comment]
Register page link is here:http://www.eventbrite.com/event/7391878307
In the registration page, We have included subscribe W3C mailing list option. We do not need add the words on the page, and twitter has been blocked by Chinese Gov, so we do not need this.

Translation:
Register = 报名注册

**Comment by lmclister** [comment]
added "register" translation.  comment out twitter stuff :/
change in at https://github.com/adobe-webplatform/testthewebfwd/commit/3fe63f47eeabbdab0c0ede71c78c2f6b25c81bbd

Zerlot, are there hash tags on weibo?  should we add that?

**Comment by zerlot** [comment]
Yes, we have a weibo account. please add "获取更多信息，请关注新浪微博@TestTheWebForward上海"

**Comment by lmclister** [comment]
added, closed
